torch/export/_trace.py

```diff
@@ -2027,6 +2027,10 @@ def _export_for_training(
 
     original_state_dict = _get_original_state_dict(mod)
 
+    # Load the functorch decomp here 
+    from torch._functorch.vmap import lazy_load_decompositions
+    lazy_load_decompositions()
```

Comment:
i remember you needed to add some code to lazily load CIA decomps into the pytorch dispatcher. should this lazy loading happen in a similar location?

Reply:
I'm confused, the first time vmap gets called the decomps will be lazily loaded. You don't need that here if that's the case, right?

Reply:
If it is non-strict export, this will be the first time we load it.